frontend/src/metabase/static-viz/components/FunnelChart/utils/funnel.ts

```diff
@@ -86,3 +87,24 @@ export const getFormattedStep = (
     stepName,
   };
 };
+
+export const reorderData = (
+  data: FunnelDatum[],
+  settings: FunnelSettings,
+): FunnelDatum[] => {
+  const funnelOrder = settings.visualization_settings["funnel.rows"];
+  if (funnelOrder == null) {
+    return data;
+  }
+
+  const keys = data.map(datum => String(datum[0]));
```

Comment:
Funnel chart key is a step name, but I need to cast it to string to avoid a situation like the **ordered funnel** chart in this comment:
https://github.com/metabase/metabase/pull/26797#issuecomment-1332343992